.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CRM Main Build
```

Comment:
Would be nice to test this, e.g. by temporarily setting the action to be run on push on any branch.

Reply:
Done, good idea, I created a Parallel PR to test that on my fork. but it is better.

Reply:
I think that it is an issue because I am not a collaborator of that repo, the parallel PR to run the CI is [that](https://github.com/femder-bdai/realsense2-lcm-driver/pull/1).

Reply:
Thanks for linking that one! All good then